### ISSUE TITLE
Big loss and always a Resource exhausted: Error

### ISSUE THREAD

**Issue by 24inch** [qqwweee/keras-yolo3#642]
Hi, 
I want to create a digit recognizer to detect sequences of handwritten digits in a video stream.
My training data consists of MNIST dataset. I created 416x416 images with 9 digits on it. Due to the fact that my loss is very big(starts with 7000!!!, after first epoche it is at around 91)
1) I do not need to train from scratch? I can use pretrained weights, right? I only have to train from scratch if I change the network?
2) When I train with yolo.h5 weights it always will stop training in last epoche and then says that I ran out of GPU mem:
`Resource exhausted: OOM when allocating tensor with shape[32,209,209,64] and type float on /job:localhost/replica:0/task:0/device:GPU:0 by allocator GPU_0_bfc`
I tried to run with batchsize:32 16 8 4 2 1, nothing worked. My GPU is a GTX1080 with 8GB.
3) I also trained the tiny-yolo, which gave me a loss of 35 in the end after 100 epochs, wich is not a good result I guess? But at least it finished training.

I checked my labels twice, they are correct.
Is there anybody who tried something similar or who ran into the same issues?

Thanks for your help and time.

**Comment by 24inch** [comment]
1) I do not have to train from start. Unless I change the network.
2) Have to reduce batchsize if all layers are unfreezed.
3) trained normal yolo to loss 30,  I can not get lower values, any tips from anyone?

Sry for the stupid questions...

**Comment by 18875140516** [comment]
I also have the problem which same with yours. I train the model on the datasets of char-detect, the loss is from 9000+ at the first epoch and down to 100+ at 40th epoch. To problem 2, it may because you didn't release the memory. I use "os.environ['CUDA_VISIBLE_DEVICES'] = '0' ", the problem appered, so then I use "K.clear_session()" after train, the problem disappered. Do you know which mistake can result in high loss?
